pkg/validation/validation.go

```diff
@@ -30,7 +31,16 @@ import (
 const gatewayClassNamespacedParamRef = "Namespace"
 
 // Contour returns true if contour is valid.
-func Contour(contour *operatorv1alpha1.Contour) error {
+func Contour(ctx context.Context, cli client.Client, contour *operatorv1alpha1.Contour) error {
+	// TODO [danehans]: Remove when https://github.com/projectcontour/contour-operator/issues/18 is fixed.
+	exist, err := objcontour.OtherContoursExistInSpecNs(ctx, cli, contour)
```

Comment:
**Note:** https://github.com/projectcontour/contour-operator/pull/282 is required for `OtherContoursExistInSpecNs()` to work properly.